Userland/Libraries/LibJS/Runtime/DeclarativeEnvironment.cpp

```diff
@@ -139,6 +143,8 @@ ThrowCompletionOr<void> DeclarativeEnvironment::set_mutable_binding(VM& vm, Depr
         // b. Perform ! envRec.CreateMutableBinding(N, true).
         MUST(create_mutable_binding(vm, name, true));
 
+        ++m_unique_environment_serial_number;
```

Comment:
This one doesn't seem necessary, since `create_mutable_bindings()` always increments the serial anyway.

Also, these serial numbers are not guaranteed to be unique, so I don't understand why we're calling them "unique" :)